h/templates/groups/share.html.jinja2

```diff
@@ -12,19 +12,24 @@
   <div class="content content--narrow">
     <div class="group-form is-member-of-group">
       <i class="h-icon-group group-form__heading-icon"></i>
-      <div class="group-form__heading">{{ group.name }}</div>
+      <div class="group-form__heading--short">{{ group.name }}</div>
 
       {% if document_links %}
-        Recently annotated documents:
-        <ul class="group-form-document-list">
+        <a href="{{ request.route_url('stream') }}?q=group:{{ group.pubid }}"
+           target="_blank"
+           title="Recent annotations in this group">
+           View recent group annotations</a>
+    </div>
```

Comment:
Thanks @robertknight ! I made some changes to make it more readable.  Note that the `{% if %}...{% endif %}` tags are outside the second div, since if there's no documents, that div shouldn't appear at all.  Let me know if there are better ways to do it!